Requests/Builders/CatalogBuilders/ChangeCatalogItemsRequestBuilder.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace PyrusApiClient.Builders
+{
+    public class ChangeCatalogItemsRequestBuilder
+    {
+        private readonly PartialUpdateCatalogRequest _request;
+        public int CatalogId { get; }
+
+        public ChangeCatalogItemsRequestBuilder(int catalogId)
+        {
+            _request = new PartialUpdateCatalogRequest();
+            CatalogId = catalogId;
+        }
+
+        public ChangeCatalogItemsRequestBuilder(PartialUpdateCatalogRequest request, int catalogId)
+        {
+            _request = request;
+            CatalogId = catalogId;
+        }
+
+        public static implicit operator PartialUpdateCatalogRequest(ChangeCatalogItemsRequestBuilder builder)
+        {
+            return builder._request;
+        }
+
+        public ChangeCatalogItemsRequestBuilder UpsertItem(CatalogItem item)
+        {
+            _request.UpsertItems.Add(item);
+            return this;
+        }
+
+        public ChangeCatalogItemsRequestBuilder UpsertItems(IEnumerable<CatalogItem> items)
+        {
+            _request.UpsertItems.AddRange(items);
+            return this;
+        }
+
+        public ChangeCatalogItemsRequestBuilder UpsertItems(params CatalogItem[] items)
+        {
+            _request.UpsertItems.AddRange(items);
+            return this;
+        }
+
+        public ChangeCatalogItemsRequestBuilder UpsertItem(IEnumerable<string> values)
+        {
+            _request.UpsertItems.Add(new CatalogItem { Values = values.ToList() });
+            return this;
+        }
+
+        public ChangeCatalogItemsRequestBuilder UpsertItem(params string[] values)
+        {
+            _request.UpsertItems.Add(new CatalogItem { Values = values.ToList() });
+            return this;
+        }
+
+        public ChangeCatalogItemsRequestBuilder DeleteItem(CatalogItem item)
```

Comment:
null check also Values has to be checked for being not null and having at least one element